micromamba/tests/test_install.py

```diff
@@ -676,6 +665,28 @@ def test_install_check_dirs(tmp_home, tmp_root_prefix):
         assert os.path.isdir(env_prefix / "lib" / "python3.8" / "site-packages")
 
 
+def test_install_local_package(tmp_home, tmp_root_prefix):
```

Comment:
Move the test outside of the class because of `C` drive access issues on windows (I also think this is the preferable way of writing tests, because it's more isolated).